idea-plugin/src/main/kotlin/arrow/meta/ide/phases/resolve/MetaSyntheticResolveExtension.kt

```diff
@@ -60,7 +60,7 @@ open class MetaSyntheticResolveExtension(project: Project) : SyntheticResolveExt
     if (!thisDescriptor.isMetaSynthetic()) {
       thisDescriptor.findPsi().safeAs<KtClassOrObject>()?.containingKtFile?.packageFqName?.let { packageName ->
         result.replaceWithSynthetics(packageName, declarationProvider)
-        cache.resolved(packageName)?.filterIsInstance<LazyClassDescriptor>()?.find { it.fqNameSafe == thisDescriptor.fqNameSafe }?.let { classDescriptor ->
+        service?.cache?.descriptors(packageName)?.filterIsInstance<LazyClassDescriptor>()?.find { it.fqNameSafe == thisDescriptor.fqNameSafe }?.let { classDescriptor ->
```

Comment:
The internal data structurs of the cache were change. Could you add a note in this PR why this was done? Is this more efficient and/or simpler code?

Reply:
Sure will do :)